swagger.yaml

```diff
@@ -5953,6 +5953,67 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/events:
```

Comment:
the events should contain also the event-id (the event name from the events.yaml) to support MGMT-6610

Reply:
per your other comment, added event_name